test/wavTest.cpp

```diff
@@ -8,11 +8,11 @@
 
 TEST(wavTest, temp){
     size_t probeSize = 1024;
-    WavHandler wav_hdr("../rsc/test3.wav");
+    WavHandler wav_hdr("../rsc/a_p.wav");
     Segment seg(probeSize);
     std::ifstream correctData;
     float data;
-    float allowedError = 0.1;
+    float allowedError = 10000;
```

Comment:
What is the reason for such increase in allowedError?

Reply:
Earlier error was relative now it is absolute

Reply:
What is the unit then?

Reply:
I don't know, but in file the smalest value is like 2*10^7 so 10^3 is pretty small compered to that

Reply:
In this case we should return to relative error

Reply:
Also in such instance it should be possible to assert to the exact value

Reply:
> Also in such instance it should be possible to assert to the exact value

I think due to the size the issue might be caused by constraints of float cause from what I remember float is inacurate for big numbers

Reply:
Changed back to relative error this time of 0.001%

Reply:
Well yes, but in this solution there is no performed operation that would introduce such inaccuracies

Reply:
maybe floats saved to file are saved in wrong format

Reply:
yeah that was that now it asserts exact value